index.html

```diff
@@ -34,13 +40,15 @@
 	<main>
 		<!-- Heading - Text -->
 		<section class="main-container">
-			<h1 class="main-title">Hey there. I'm Hernán</h1>
-			<h2 class="main-subtitle">I'am a Software Developer</h2>
-			<p class="main-text">I can help you build a product , feature or website Look through some of my work and
-				experience!
-				If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+			<div class="text-container">
+				<h1 class="main-title">Hey There. I'm Hernán</h1>
+				<h2 class="main-subtitle">I'am a Software Developer</h2>
+				<p class="main-text">I can help you build a product , feature or website Look through some of my work and
+					experience!
+					If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+			</div>
```

Comment:
- [ ] Well done designing the desktop structure of this section however, you missed to add a background image as shown below 🔽 

### Figma Design
![Figma header](https://user-images.githubusercontent.com/104319462/220785024-33bf0949-3d91-4ec1-b056-989fe1b2b4d3.png)

### Your Design
![user header design](https://user-images.githubusercontent.com/104319462/220785044-a157784f-1e14-4243-9e50-c4734a0d554d.png)